src/tests/unittests/instrument_drivers/virtualAwg/test_virtual_awg.py

```diff
@@ -15,7 +16,7 @@ def tearDown(self) -> None:
         self.settings.close()
 
     @staticmethod
-    def __create_awg_drivers() -> List[Mock]:
+    def __create_awg_drivers():
```

Comment:
Why remove the type annotation? If there are warnings of errors, can we not relax to something like `List[Any]`?

Reply:
Type hinting unit tests is asking for mypy errors because in the unit tests we use Mock classes when calling production code instead of the production classes. But indeed, List[Any] is possible here.